blueprints/app/files/.travis.yml

```diff
@@ -9,6 +9,17 @@ cache:
   directories:
     - node_modules
 
+env:
+  - E_SCENARIO=ember-1.10
```

Comment:
I think we should stick to just checking the channels and the "default" (which may require some tweaks in ember-try).

Also, can we make the environment variable `EMBER_TRY_SCENARIO` (to me a bit more explicit)?